src/bucket.ts

```diff
@@ -340,6 +340,7 @@ class Bucket extends ServiceObject {
       id: name,
       createMethod: storage.createBucket.bind(storage),
       methods,
+      requestModule: request,
```

Comment:
Is this strictly required?  For each library, does that mean we need to go pull in request now?

Reply:
@fhinkel

Reply:
Yes. For now, instead of `common`, each client library needs to provide an HTTP dependency. Of course, if not needed, you can provide an empty object here.